mithril-common/src/test_utils/fixture_builder.rs

```diff
@@ -279,4 +311,67 @@ mod tests {
             assert!(!first_party_ids.contains(&party_id));
         }
     }
+
+    /// Verify that there is cached kes key for a number of party id.
+    /// If the cache is not up to date, the test will generate the code that could be copy/paste into the [precomputed_keskey] module.
```

Comment:
If the file is renamed:
```suggestion
    /// If the cache is not up to date, the test will generate the code that can be copied and pasted into the [precomputed_kes_key] module.
```

Reply:
Done